geode-lucene/src/distributedTest/java/org/apache/geode/cache/lucene/ExpirationDUnitTest.java

```diff
@@ -76,13 +75,7 @@ public void regionWithExpirationSetMustAlsoRemoveLuceneIndexEntries(
           LuceneQuery<Integer, TestObject> luceneQuery = luceneService.createLuceneQueryFactory()
               .setLimit(100).create(INDEX_NAME, REGION_NAME, "world", "text");
 
-          Collection luceneResultList = null;
-          try {
-            luceneResultList = luceneQuery.findKeys();
-          } catch (LuceneQueryException e) {
-            e.printStackTrace();
-            fail();
-          }
+          Collection luceneResultList = luceneQuery.findKeys();
```

Comment:
I know this is a usage of Awaitility, but it was near a usage that I reviewed and I couldn't help fixing this block of code.